### PR TITLE
Add recommendations from Cantina

### PR DESCRIPTION
**Pull request by telome**

### CODE REVIEW

File: src/L2TokenGateway.sol [hunk 1]
```diff
@@ -163,14 +163,14 @@ contract L2TokenGateway is ITokenGateway, ICustomGateway, L2ArbitrumMessenger {
...
-            ITokenGateway.finalizeInboundTransfer.selector,
+        outboundCalldata = abi.encodeCall(
+            ITokenGateway.finalizeInboundTransfer, (
             token,
             from,
             to,
             amount,
             abi.encode(0, data) // using 0 for exitNum as exit redirection is not supported
-        );
+        ));
```
Comment (sunbreak1211) [inline src/L2TokenGateway.sol:172]: I'd suggest doing the named params for sendTxToL1 as we did in the other side Gateway.

Comment (sunbreak1211) [reply src/L2TokenGateway.sol:172]: I mean I'd say we do on both or none.

Comment (telome) [reply src/L2TokenGateway.sol:172]: Yes, done.

**Review by sunbreak1211 — APPROVED** [review]
LGTM

**Review by oldchili — APPROVED** [review]
LGTM

**Pull request merged**